app/decorators/drawing_decorator.rb

```diff
@@ -16,7 +16,7 @@ def story_concat
 
   private
 
-  def text_concat(text, max_length=30)
-    text[0, max_length - 1].concat('...')
+  def text_concat(text, max_length=100)
```

Comment:
The things is, the image sets the height of each row quite high in most cases. If you see in the screenshot, even 100 characters is not filling all the space. If we have the space reserved already because of the row height, we may as well make use of it if it'll reduce the need for user to click through the the show view